usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -406,6 +428,28 @@ This endpoint returns a list of data that is associated with the award profile p
                     "foreign_province": null,
                     "foreign_postal_code": null
                 },
+                "total_account_outlay": 3.0,
+                "total_account_obligation": 3.0,
+                "account_outlays": {
+                    {
+                        "code": "P",
+                        "amount": 2.0
+                    },
+                    {
+                        "code": "L",
+                        "amount": 1.0
+                    }
+                },
+                "account_obligations": {
```

Comment:
Ohhhh... so close... these are returning an array of objects... not an object of objects  😛